mirrors.yaml

```diff
@@ -1,404 +1,711 @@
+  - 46.5969028, 125.1014811
+  - 45.7593633, 126.6276177
+  - 47.328863, 130.2816917
+  - 49.249879, 127.11731
+  - 47.02157, 132.044137
+  - 46.880466, 126.808707
+  - 34.7487244, 113.6191483
+  - 30.5951051, 114.2999353
+  - 28.1450774, 113.2384362
+  - 29.3594838, 113.1229551
+  - 40.6538754, 109.8729897
   - 40.8337963, 111.6730788
-  - 22.2793278, 114.1628131
-  - 23.1146515, 114.4107202
-  - 50.4500336, 30.5241361
-  - 39.5128345, 116.6997446
-  - 51.5073219, -0.1276474
-  - 40.4167047, -3.7035825
-  - -37.8142176, 144.9631608
-  - 45.5074301, 9.2446893
-  - 55.7504461, 37.6174943
-  - 19.0759899, 72.8773928
-  - 32.0437711, 118.7788734
-  - 48.8206809, 2.3635646
+  - 43.6165326, 122.2601837
+  - 31.8122623, 119.9691539
+  - 32.0615513, 118.7915619
+  - 31.9827896, 120.8904588
+  - 31.5776626, 120.2952752
+  - 33.349559, 120.1577019
+  - 32.3968554, 119.4077658
+  - 29.6653828, 115.9474861
+  - 28.6472124, 116.0348483
+  - 43.8844201, 125.3180998
+  - 39.740278, 122.255833
+  - 41.8026095, 123.4279105
+  - 36.824463, 101.440811
+  - 34.2607713, 108.9423443
+  - 36.6507007, 117.1140042
   - 36.0638034, 120.3781372
-  - 37.3361905, -121.890583
-  - 47.6038321, -122.3300624
-  - 31.2322758, 121.4692071
-  - 22.555454, 114.0543297
-  - 38.0429742, 114.5088385
+  - 37.461928, 121.4425255
+  - 31.2312707, 121.4700152
+  - 37.6866796, 112.7383414
+  - 37.870009, 112.5452739
+  - 30.6598628, 104.0633717
+  - 39.1175488, 117.1913008
+  - 43.419754, 87.319461
+  - 25.0363006, 102.7086139
+  - 30.2489634, 120.2052342
+  - 28.6581723, 121.4163876
+  - 48.8534951, 2.3483915
+  - 50.1106444, 8.6820917
+  - 22.2793278, 114.1628131
+  - 28.6273928, 77.1716954
+  - 18.9733536, 72.82810491917377
+  - -6.175247, 106.8270488
+  - 45.4641943, 9.1896346
+  - 34.6198813, 135.490357
+  - 35.6840574, 139.7744912
+  - 3.1516964, 101.6942371
+  - 52.3730796, 4.8924534
+  - 6.4550575, 3.3941795
+  - 14.5906346, 120.9799964
+  - 55.7505412, 37.6174782
   - 1.357107, 103.8194992
-  - 39.0017455, -77.4033349
+  - -26.205, 28.049722
+  - 37.5666791, 126.9782914
+  - 40.4167047, -3.7035825
   - 59.3251172, 18.0710935
-  - 31.3016935, 120.5810725
-  - -33.8548157, 151.2164539
-  - 39.0856735, 117.1951073
-  - 35.6828387, 139.7594549
-  - 30.5951051, 114.2999353
-  - 31.4933074, 120.3074357
-  - 33.3495657, 120.1576778
-  - 32.3969935, 119.4077008
-  - 22.5213807, 113.3656141
+  - 25.061706, 121.4589414
+  - 25.0375198, 121.5636796
+  - 13.8245796, 100.6224463
+  - 41.006381, 28.9758715
+  - 50.4500336, 30.5241361
+  - 51.4893335, -0.14405508452768728
+  - 37.3361663, -121.890591
+  - 39.7392364, -104.984862
+  - 25.7741728, -80.19362
+  - 41.8755616, -87.6244212
+  - 40.7127281, -74.0060152
+  - 32.7762719, -96.7968559
+  - 39.0017455, -77.4033349
+  - 47.6038321, -122.330062
+  - 10.7763897, 106.7011391
 - provider: 华为云
   url: https://repo.huaweicloud.com/archlinuxcn/
-  added_date: 2020-10-31
+  comment: It may fail to synchronize for a long time
```

Comment:
@lilydjwg Once the mirror is normal again, please delete this comment and run the script to regenerate the file.